### DISCUSSION TITLE
Vim mode

### DISCUSSION THREAD

**Discussion by cowboy8625** [ndd7xv/heh#157] [Ideas]
Any plans on a vi keybind mode?  I do see there are some already by default but it would be nice if there was more.  Also if your not wanting to do this would you be apposed to a PR with the changes?

**Comment by ndd7xv** [comment]
Definitely something I would look into if I had the time, which might be the case next year. Regardless of if I get to it or not, I always welcome interest and PRs, so I would 100% not be opposed :)

**Reply by cowboy8625** [reply]
Time is always a key factor!  Well regardless thanks for the application I use it quit often.  I may give it a shot but no promises.

**Reply by orhun** [reply]
I would be happy to also implement it soon if you can clarify what kind of Vim key bindings you want. HJKL is already there :)

**Reply by cowboy8625** [reply]
`wjkl` worka yes but they are not behide a mode.  `w` does not work and when pressing `b` it replaces the character under cursor to `B` which is a bit confusing if `wjkl` works on the same layer.  I would expect a `normal` mode for movement and `insert` mode for editing. After searching pressing `n` or `p` does not work unless pressing `ctrl`.
